### DISCUSSION TITLE
Highlight one author

### DISCUSSION THREAD

**Discussion by fl-yu** [alshedivat/al-folio#609] [Q&A]
Is there possible to highlight a particular author in the publications with bold, regardless of whether it is the first author or not? Thank you so much!!!

**Comment by mbarbetti** [comment]
Yes, but you need to code a bit. I think that the simplest solution it to add the author name to the `coauthors.yml` file (you can find it within `_data`) and then edit properly the `bib.html` file (you can find it within `_layouts`), for instance the following lines:
```html
{% if author_is_self %}
    and <em>{{author.first}} {{author.last}}</em>
{% else %}
    {% if coauthor_url %}
        and <a href="{{coauthor_url}}" target="_blank">{{author.first}} {{author.last}}</a>
    {% else %}
        and {{author.first}} {{author.last}}
    {% endif %}
{% endif %}
```
An alternative (and more elegant) solution is to edit correctly the `_base.scss` file (you can find it within `_sass`), but I am not a Ruby expert so I can help you only partially.
